packages/contracts-bedrock/src/dispute/FaultDisputeGame2.sol

```diff
@@ -240,8 +245,8 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         if (stepPos.depth() != MAX_GAME_DEPTH + 1) revert InvalidParent();
 
         // Determine the expected pre & post states of the step.
-        Claim preStateClaim;
-        ClaimData storage postState;
+        Claim postStateClaim_;
```

Comment:
I guess that following the current coding style, the local variables should not end with "_".